debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerStreamingChangeEventSource.java

```diff
@@ -191,15 +193,28 @@ public void execute(ChangeEventSourceContext context) throws InterruptedExceptio
                                 tableWithSmallestLsn.next();
                                 continue;
                             }
+
+                            if (tableWithSmallestLsn.isNewTransaction() && changesStoppedBeingMonotonic.get()) {
+                                LOGGER.info("Resetting changesStoppedBeingMonotonic as transaction changes");
```

Comment:
That's more for DEBUG level perhaps? Or is this a rare thing to happen?

Reply:
IIUC it should happen only once upon start then I'd keep it as info,

Reply:
Ok, makes sense.